fiftyone/server/aggregations.py

```diff
@@ -42,6 +42,7 @@ class AggregationForm:
     slices: t.Optional[t.List[str]]
     view: BSONArray
     view_name: t.Optional[str] = None
+    query_performance: t.Optional[bool] = False
```

Comment:
where does this value come from?/how is it set?

Reply:
It comes from the App setting `@fiftyone/state/queryPerformance` atom